src/smpi/smpi_global.cpp

```diff
@@ -96,6 +97,12 @@ static char *get_mailbox_name_small(char *str, int index)
   return str;
 }
 
+static char *get_mailbox_name_migration(char *str, int index)
+{
+  snprintf(str, MAILBOX_NAME_MAXLEN, "mig-%0*x", static_cast<int> (sizeof(int) * 2), index);
```

Comment:
This was done like this to use the same naming scheme of the other two already existing mailboxes.